lib/nimble.phx.gen.template/project.ex

```diff
@@ -7,4 +9,17 @@ defmodule Nimble.Phx.Gen.Template.Project do
             web_path: nil,
             web_test_path: nil,
             api_project?: nil
+
+  def info(opts \\ %{}) do
```

Comment:
Controversial as it echoes OOP, but this method could also be named `new(opts \\ %{})` as it returns the Project module as a map.